src/autopas/containers/CellBasedParticleContainer.h

```diff
@@ -32,11 +32,17 @@ class CellBasedParticleContainer : public ParticleContainerInterface<typename Pa
    * @param boxMin
    * @param boxMax
    * @param cutoff
-   * @param skin
+   * @param skinPerTimestep
+   * @param rebuildFrequency
    */
   CellBasedParticleContainer(const std::array<double, 3> boxMin, const std::array<double, 3> boxMax,
-                             const double cutoff, const double skin)
-      : _cells(), _boxMin(boxMin), _boxMax(boxMax), _cutoff(cutoff), _skin(skin) {}
+                             const double cutoff, const double skinPerTimestep, const unsigned int rebuildFrequency)
+      : _cells(),
+        _boxMin(boxMin),
+        _boxMax(boxMax),
+        _cutoff(cutoff),
+        _skinPerTimestep(skinPerTimestep),
+        _rebuildFrequency(rebuildFrequency) {}
```

Comment:
why change this? Isn't it simpler to calculate the skin in the high-level interface and pass down the calculated skin value here? (applies also to subsequent files)

Reply:
Good Question! I wasn't sure how "deep" we wanted to carry this through.

Reply:
Should be fixed now. Please let me know if you would prefer to use a different strategy instead.